plugin.tmux

```diff
@@ -28,9 +28,12 @@ main() {
   done
 
   local -r opt_key="$(get_tmux_option "@1password-key" "u")"
+  local -r clear_key="$(get_tmux_option "@1password-key" "C")"
 
   tmux bind-key "$opt_key" \
     run "tmux split-window -l 10 \"$CURRENT_DIR/scripts/main.sh '#{pane_id}'\""
+
+  tmux bind-key "$clear_key" run "$CURRENT_DIR/scripts/clear.sh"
```

Comment:
Urghhh the `clear.sh` has a duplicated logic as in `main.sh`. I suggest making this line as:

```suggestion
  tmux bind-key "$clear_key" run "$CURRENT_DIR/scripts/main.sh clear-cache"
```

And then, in `main.sh`, check the value of the given argument. If it is `"clear-cache"`, then clear the cache. Else, run the usual flow.